cachito/workers/scm.py

```diff
@@ -147,7 +167,23 @@ def fetch_source(self):
             ) as exc:
                 log.warning("Error handling archived artifact '%s': %s", previous_archive, exc)
 
-        self.clone_and_archive()
+        self.clone_and_archive(gitsubmodule)
+
+    def update_git_submodule(self, repo):
```

Comment:
(optional): Should this be `update_git_submodules` (plural) since we expect there could be more than one?